src/wallet/mod.rs

```diff
@@ -440,7 +439,11 @@ where
         builder.ordering.sort_tx(&mut tx);
 
         let txid = tx.txid();
-        let psbt = self.complete_transaction(tx, prev_script_pubkeys, builder)?;
+        let lookup_output = selected
+            .into_iter()
+            .map(|utxo| (utxo.outpoint, utxo))
+            .collect();
+        let psbt = self.complete_transaction(tx, lookup_output, builder)?;
```

Comment:
❓ @afilini what do you think about just building this hashmap inside complete transaction.

Reply:
If you can find a nice way to do it, absolutely: IIRC I tried doing that while I was refactoring something but I was having some troubles in `bump_fee` because the `selected` vec only contained the extra added inputs, not all the inputs of the transaction, and that was breaking something in `complete_transaction`.

Reply:
I did this. I don't see what you mean because the hashmap originally only had the "extra added inputs".

Reply:
Ah then I guess I don't remember where I got stuck the last time :joy: 

Well, glad you managed to make it work though, that's the important thing!